compute_tools/src/compute.rs

```diff
@@ -132,6 +134,10 @@ pub struct ComputeState {
     /// Timestamp of the last Postgres activity. It could be `None` if
     /// compute wasn't used since start.
     pub last_active: Option<DateTime<Utc>>,
+    /// Timestamp of the last client's activity. Unlike `last_time` it doesn't take in account
+    /// baclkground activity: autovacuum, LR,...
+    pub last_active_query: Option<DateTime<Utc>>,
```

Comment:
Since you modify the compute response, you also need to modify the OpenAPI spec https://github.com/neondatabase/neon/blob/5819938c93f2eb504b0e7593314d6d1f168c6196/compute_tools/src/http/openapi_spec.yaml#L414

It's not currently used to generate HTTP clients, but it's a good reference. You can use https://editor.swagger.io/ or your IDE plugin to verify that it's valid

Reply:
Thank you for noticing it!
Done.